nebullvm/operations/optimizations/compilers/openvino.py

```diff
@@ -25,6 +25,137 @@
 from nebullvm.tools.transformations import MultiStageTransformation
 
 
+# classe di base. Contains generic methods.
+class TensorFlowOpenVINOCompiler(Compiler):
+    supported_ops = {
+        "cpu": [
+            None,
+            # QuantizationType.STATIC,
+            # QuantizationType.HALF,
+        ],
+        "gpu": [],
+    }
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(
+        self,
+        model: Union[str, Path],
```

Comment:
the input for tensorflow compilers should be a tensorflow mode not a file. You should then save it to file and just at the end give it as input to the OpenVino CLI.

Reply:
Sure, this is how I would do it:

1. save the model to a file: 
`model.save('saved_model/my_model')`

2. Load the model:
new_model = tf.keras.models.load_model('saved_model/my_model')

3. Pass it as input to the OpenVino CLI
`mo --input_model <INPUT_MODEL>.pb`

Reply:
You can skip step 2, the execute method receives a tensorflow model, you can then save it to file (remember to save it to a temp directory as we do in the other sections) and pass it to the `mo` command